### PR TITLE
Add testids

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Add test-ids attributes.

#### What problem is this solving?
Lack of reference points for e2e tests.

#### How should this be manually tested?
`yarn styleguide`

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Totalizer/TotalizerLabel.js [hunk 1]
```diff
@@ -2,7 +2,11 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 const TotalizerLabel = ({ label, mobileScroll }) => (
-  <div className={`c-muted-2 f6 ${mobileScroll ? 'nowrap' : ''}`}>{label}</div>
+  <div
+    data-testid="totalizer-label"
+    className={`c-muted-2 f6 ${mobileScroll ? 'nowrap' : ''}`}>
```
Comment (emersonlaurentino) [inline react/components/Totalizer/TotalizerLabel.js:7]: could you use `classnames`?

File: react/components/Totalizer/TotalizerValue.js [hunk 2]
```diff
@@ -23,7 +23,9 @@ class TotalizerValue extends PureComponent {
     }
 
     return (
-      <div className={`f4 fw5 c-on-base ${mobileScroll ? 'nowrap' : ''}`}>
+      <div
+        data-testid="totalizer-value"
+        className={`f4 fw5 c-on-base ${mobileScroll ? 'nowrap' : ''}`}>
```
Comment (emersonlaurentino) [inline react/components/Totalizer/TotalizerValue.js:28]: Also here

File: react/components/Totalizer/TotalizerLabel.js (hunk 1 above)
Comment (kevinch) [reply react/components/Totalizer/TotalizerLabel.js:7]: Good point.

File: react/components/Totalizer/TotalizerValue.js (hunk 2 above)
Comment (kevinch) [reply react/components/Totalizer/TotalizerValue.js]: Updated.

File: react/components/Totalizer/TotalizerLabel.js (hunk 1 above)
Comment (kevinch) [reply react/components/Totalizer/TotalizerLabel.js]: Updated.

**Pull request merged**